lib/grpcServer/handlers/platform/getConsensusParamsHandlerFactory.js

```diff
@@ -0,0 +1,82 @@
+const InvalidArgumentGrpcError = require('@dashevo/grpc-common/lib/server/error/InvalidArgumentGrpcError');
+
+const {
+  server: {
+    error: {
+      InternalGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+const {
+  v0: {
+    GetConsensusParamsResponse,
+    ConsensusParamsBlock,
+    ConsensusParamsEvidence,
+  },
+} = require('@dashevo/dapi-grpc');
+
+const RPCError = require('../../../rpcServer/RPCError');
+const InvalidHeightError = require('../../../errors/InvalidHeightError');
+
+/**
+ *
+ * @param {getConsensusParams} getConsensusParams
+ * @returns {getConsensusParamsHandler}
+ */
+function getConsensusParamsHandlerFactory(getConsensusParams) {
+  /**
+   * @typedef getConsensusParamsHandler
+   * @param {Object} call
+   * @returns {Promise<>}
+   */
+  async function getConsensusParamsHandler(call) {
+    const { request } = call;
+
+    const prove = request.getProve();
+
+    if (prove) {
+      throw new InvalidArgumentGrpcError('Prove is not implemented yet');
+    }
+
+    const height = request.getHeight();
+
+    let consensusParams;
+
+    try {
+      consensusParams = await getConsensusParams(height);
+    } catch (e) {
+      if (e instanceof RPCError) {
+        if (e.code === 32603) {
+          throw new InvalidHeightError(e.data, null, e.code);
```

Comment:
Actually you need to throw gRPC error with gRPC error code. Precondition Failed is the best candidate.